src/python/pants/backend/codegen/tasks/simple_codegen_task.py

```diff
@@ -78,6 +103,31 @@ def codegen_targets(self):
     """
     return self.context.targets(self.is_gentarget)
 
+  @classmethod
+  def _codegen_strategy_for_name(cls, name):
+    return {
+      'global': SimpleCodegenTask.GlobalCodegenStrategy,
```

Comment:
1) This is a classmethod, so you could write: `cls.GlobalCodegenStrategy`
2) I think I would have just made this a class constant instead of a method.  Are you doing it this way to keep the data from being modified or some other reason?